src/components/field-array-source/field-array-source.component.tsx

```diff
@@ -0,0 +1,88 @@
+import { Field, FieldArray, FieldProps, Form, Formik, getIn } from 'formik';
+import * as React from 'react';
+import { generate } from 'shortid';
+import * as yup from 'yup';
+import { localization } from '../../lib/localization';
+import { InputField } from '../../components/field-input/field-input.component';
+import { TextAreaField } from '../../components/field-textarea/field-textarea.component';
+import { ButtonNewSource } from '../button-new-source/button-new-source.component';
+import { ButtonRemoveSource } from '../../components/button-remove-source/button-remove-source.component';
+import { SelectField } from '../../components/field-select/field-select.component';
+import { v4 } from 'uuid';
+import './field-array-source.scss';
+
+interface Props {
+  form: {
+    values: {
+      kildebeskrivelse: {
+        kilde: any[];
+      };
+    };
+  };
+}
+
+const options = [
```

Comment:
bruk localization i stedet for hard-kodete verdier

Reply:
Ok